### PR TITLE
Fix/bugs from excel

### PR DESCRIPTION
**Pull request by theketan26**
## Description

Bug fixes from excel sheet

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [ ] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

**Comment by punitkashyup** [comment]
Please attach the issue link

File: src/packages/@app/pages/RestExplorer/RestExplorerPage.ViewModel.ts [hunk 1]
```diff
@@ -443,6 +447,11 @@ class RestExplorerViewModel
     this.updateRequestState({ isSendRequestInProgress: true });
     const start = Date.now();
 
+    this._tab
+      .subscribe((tab) => {
+        tab.property.request.url = tab.property.request.url.trim();
+      })
+      .unsubscribe();
     const decodeData = this._decodeRequest.init(
```
Comment (itsmdasifraza) [inline src/packages/@app/pages/RestExplorer/RestExplorerPage.ViewModel.ts:455]: do this in decode-request.ts

File: src/packages/@common/components/tooltip/Tooltip.svelte [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  afterUpdate(() => {
+    getPosition();
+  });
+
+  $: {
+    if (windowHeight || windowWidth) {
+      getPosition();
+    }
+  }
+</script>
```
Comment (itsmdasifraza) [inline src/packages/@common/components/tooltip/Tooltip.svelte:34]: move this file to library/ui

File: src/packages/@common/utils/init-folder-tab.ts [hunk 3]
```diff
@@ -73,6 +74,9 @@ class InitFolderTab {
   public updateIsSave(_isSave: boolean) {
     this._tab.isSaved = _isSave;
   }
+  public updateTotalRequests(_requests: number) {
+    this._tab.property.totalRequests = _requests;
+  }
```
Comment (itsmdasifraza) [inline src/packages/@common/utils/init-folder-tab.ts:79]: not required

**Comment by punitkashyup** [comment]
Please attach the issue link

**Comment by punitkashyup** [comment]
Please attach the issue link

**Review by itsmdasifraza — APPROVED** [review]

**Review by gc-codes — APPROVED** [review]

**Pull request merged**